docs/installation.md

```diff
@@ -1,4 +1,50 @@
-# Run the Helm Broker locally
+# Installation 
+
+This document provides two instructions on the Helm Broker local installation. Read [this](#install-helm-broker-from-chart) section to learn how to install the Helm Broker from the chart together with the Service Catalog, and [this](#install-helm-broker-manually) section to learn how to install the Helm Broker manually as a standalone component.
+
+## Prerequisites
+
+
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 1.16
+* [Helm CLI](https://github.com/kubernetes/helm#install) 2.14
+* [Docker](https://docs.docker.com/install/) 19.03 
+* [Kind](https://github.com/kubernetes-sigs/kind#installation-and-usage) 0.5
+
+>**NOTE:** For non-local installation, use Kubernetes v1.15.
+
+## Install Helm Broker from chart
+
+To run the Helm Broker, you need a Kubernetes cluster with Tiller and Service Catalog. Run the `./hack/run-dev-kind.sh` script, or follow these steps to set up the Helm Broker on Kind with all necessary dependencies:
```

Comment:
I don't know what the right convention is but this command originally `./hack/run-dev-kind.sh` was in root dir, for know run this command from docs directory has no sense. Should we mention that command should be run from main directory or from the docs point of view this doesn't matter?

Reply:
@jasiu001 We assume that the user is on the root in his terminal window, not in the `docs` folder.